docs/ref-fleet-yaml.md

```diff
@@ -250,3 +250,43 @@ overrideTargets:
         env: dev
 
 ```
+
+### Helm Options
+
+#### How fleet-agent deploys the bundle
+
+These options also apply to kustomize- and manifest-style bundles.
+They control how the fleet-agent deploys the bundle. All bundles are converted into Helm charts and deployed with the Helm SDK.
+These options are often similar to the Helm CLI options for install and update.
+
+* releaseName
+* takeOwnership
+* force
+* atomic
+* disablePreProcess
+* disableDNS
+* skipSchemaValidation
+* waitForJobs
+
+#### Helm Chart Download Options
+
+These options are for Helm style bundles, they specify how to download the chart.
+
+* chart
+* repo
+* version
+
+The reference to the chart can be either:
+* a local path in the cloned Git repository, specified by `chart`.
+* a go-getter URL, specified by `chart`. This can be used to download a tarball
```

Comment:
Would it make sense to add a link here, for users not familiar with go-getter? Eg:
```suggestion
* a [go-getter URL](https://github.com/hashicorp/go-getter?tab=readme-ov-file#url-format), specified by `chart`. This can be used to download a tarball
```